report-extensions.js

```diff
@@ -36,18 +36,21 @@ function sortedKeys(s) {
     const notInOpen = Object.keys(stat.notInOpen).length;
     const notInMS = stat.notInMS.length;
     const total = upToDate + notInOpen + outdated + unstable + notInMS;
+    const failedSum = unstable + outdated;
```

Comment:
Failder are one which build failed, unstable we don’t even build, it is one for which we publish something what is not published to msmarketplace

Reply:
Just: stat.failed.length